RNTrueLayerPaymentsSDK/js/NativeTrueLayerPaymentsSDK.ts

```diff
@@ -18,7 +18,52 @@ interface Spec extends TurboModule {
   /**
    * ReactNative raw interface. Do not use directly. Use TrueLayerPaymentsSDKWrapper class instead.
    */
-  _configure(environment: string): Promise<void>;
+  _configure(
+    environment: string,
+    theme?: {
+      android?: {
+        lightColors?: {
+          primary?: string;
+          background?: string;
+          onBackground?: string;
+          surface?: string;
+          onSurface?: string;
+          error?: string;
+        };
+        darkColors?: {
+          primary?: string;
+          background?: string;
+          onBackground?: string;
+          surface?: string;
+          onSurface?: string;
+          error?: string;
+        };
+        typography?: {
+          bodyLarge?: {
+            font?: string
+          };
+          bodyMedium?: {
+            font?: string
+          };
+          bodySmall?: {
+            font?: string
+          };
+          titleLarge?: {
+            font?: string
+          };
+          titleMedium?: {
+            font?: string
+          };
+          headlineSmall?: {
+            font?: string
+          };
+          labelLarge?: {
+            font?: string
+          };
+        }
```

Comment:
This is not acceptable in a function signature. Please extract all that content into a dedicated object and just mention the object here.

Reply:
You did that in the wrapper, no?

Reply:
Yeah but using the object here throws an `UnsupportedTypeScriptGenericParserError` so all the other function signatures in that file are structured like this even though there's an object in the wrapper. If you know a way around it let me know.